src/services/expenses.service.js

```diff
@@ -0,0 +1,69 @@
+const { data } = require('../data');
+
+const getAllExpenses = () => {
+  return data.expenses;
+};
+
+const getById = (id) => {
+  return data.expenses.find((exp) => exp.id === parseInt(id));
```

Comment:
When comparing the 'id' parameter with 'exp.id', you are using the 'parseInt' function to convert 'id' to a number. However, if 'id' is not a number, 'parseInt' will return 'NaN', which could lead to unexpected behavior. It would be better to first check if 'id' can be converted to a number and handle the case when it cannot.